tests/product/standalone/presto_installer.py

```diff
@@ -116,12 +116,8 @@ def copy_presto_rpm_to_master(self, cluster=None):
     @staticmethod
     def _download_rpm():
         rpm_filename = 'presto-server-rpm.rpm'
-        rpm_path = os.path.join(prestoadmin.main_dir,
-                                rpm_filename)
-        urllib.urlretrieve(
-            'https://repository.sonatype.org/service/local/artifact/maven'
-            '/content?r=central-proxy&g=com.facebook.presto'
-            '&a=presto-server-rpm&e=rpm&v=RELEASE', rpm_path)
+        subprocess.check_call(['(cd ' + prestoadmin.main_dir +
```

Comment:
Why not just make this a dependency in the Makefile and error out here.
Putting this here hides the dependency relationship.
If we've gotten here without the dependency being satisfied, we should fail.